src/rgw/rgw_lc_s3.h

```diff
@@ -258,5 +327,5 @@ class RGWLifecycleConfiguration_S3 : public RGWLifecycleConfiguration, public XM
   void dump_xml(Formatter *f) const;
 };
 
-
+bool check_date(const string& _date);
```

Comment:
this doesn't need to be exposed in the header if it's only used locally in rgw_lc_s3.cc